### DISCUSSION TITLE
github.io coming twice in page link

### DISCUSSION THREAD

**Discussion by ritesh-ojha** [community/community#46735] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I make my portfolio website so I got this link (https://ritesh-ojha.github.io/Portifolio.github.io/) but I see others have only (Portifolio.github.io)  only can you tell me why this happening. ?

**Comment by yoannchaudet** [comment]
Every GitHub user is given a "user" site in the form `<GitHub handle>.github.io`. Because your handle is `ritesh-ojha` yours is `ritesh-ojha.github.io`. If you rename your repository from `Portifolio.github.io` to `ritesh-ojha.github.io`, your site will be served directly at `https://ritesh-ojha.github.io`.

You can rename your GitHub account if you want to change your default Pages URL. But at this point I would suggest setting up a custom domain instead.

You can learn more about the two kinds of Pages sites here: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites.

**Reply by occhem** [reply, marked as answer]
@ritesh-ojha are you able to solve the problem? please let me know. thank you

**Comment by alexzoo0** [comment]
I have the same problem (06/12/2025) I think I has to do with the way you set up the DNS and "Homepage" in package.json. If I take off the custom website in github pages and set the "Homepage":"https://USERNAME.github.io/REPOSITORY/" it won't show twice but if I add the custom website and set up your homepage like this  "Homepage":"https://USERNAME.github.io/", it'll show twice.
